config/settings-production.yml

```diff
@@ -80,6 +82,7 @@ buffy:
         - reviewers-list
       remove_labels:
         - 5/awaiting-reviewer(s)-response
+        - 4/review(s)-in-awaiting-changes
```

Comment:
But this looks to me like it should work:
https://github.com/ropensci-org/buffy/blob/044d630b554a7963aa2a1d8febe9422edd5643bd/app/responders/label_command_responder.rb#L26